generators/base-core/generator.ts

```diff
@@ -68,6 +68,7 @@ import { getGradleLibsVersionsProperties } from '../gradle/support/dependabot-gr
 import { dockerPlaceholderGenerator } from '../docker/utils.js';
 import { getConfigWithDefaults } from '../../jdl/index.js';
 import { extractArgumentsFromConfigs } from '../base/internal/command.js';
+import { JSONGeneratorJhipsterContent } from '../../jdl/converters/types.js';
```

Comment:
We should not mix jdl with generator types.
To add a type to jhipsterConfig a new type should be created join storage types from every generator.